src/main/webapp/IFrame.js

```diff
@@ -28,23 +28,23 @@ const vidDataRef = firestore.collection('rooms').doc(roomId)
 const queueDataRef = firestore.collection('rooms').doc(roomId)
     .collection('queue');
 
-let videoIds;
-let thumbnails;
-let docIds;
+let nextVidID;
+let nextThumbnail;
+let nextDocID;
 function updateQueue() {
-  videoIds = [];
-  thumbnails = [];
-  docIds = [];
-  queueDataRef.orderBy('requestTime', 'asc').get()
+  nextVidID = '';
+  nextThumbnail = '';
+  nextDocID = '';
+  queueDataRef.orderBy('requestTime', 'asc').limit(1).get()
       .then(function(querySnapshot) {
         querySnapshot.forEach(function(doc) {
           const queueData = doc.data();
-          docIds.push(doc.id);
-          videoIds.push(queueData.videoID);
+          nextDocID += doc.id;
+          nextVidID += queueData.videoID;
```

Comment:
Doesn't this concatenate the video ID? Shouldn't it just be 
```

nextDocID = doc.id;
 nextVidID = queueData.videoID;
```

Reply:
Yeah, my mistake, I thought I had to concatenate to have the empty string be the default, but I now see that just setting it equal would work too the way I have it set up.